lua/wire/wire_paths.lua

```diff
@@ -42,14 +42,6 @@ if CLIENT then
 		path.Entity.WirePaths[path.Name] = path
 
 	end)
-
-	hook.Add("NetworkEntityCreated", "WireLib.Paths.NetworkEntityCreated", function(ent)
-		if ent.Inputs then
-			net.Start("WireLib.Paths.RequestPaths")
-			net.WriteEntity(ent)
-			net.SendToServer()
-		end
-	end)
```

Comment:
As far as I can see this is the only place where `WireLib.Paths.RequestPaths` messages are sent, so now we never request paths. If that's not the case, where's it called from? I can't find it.

If it is supposed to be removed, we should remove the receiving code as well. How are clients supposed to receive the existing paths when they join a server?